### PR TITLE
add: `linked-deb`

### PR DESCRIPTION
**Pull request by soumyatheman**

### CODE REVIEW

**Review by Henryws — CHANGES_REQUESTED** [review]
Add to `packagelist`

**Comment by soumyatheman** [comment]
What is packagelist?

**Comment by soumyatheman** [comment]
Done

File: packages/linked-deb/linked-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name="linked-deb"
+gives="linked"
+version="1.4.13"
+replace="${gives}"
+url="https://github.com/lostdesign/linked/releases/download/v${version}/linked_${version}_amd64.deb"
+description="An easy, distraction-free way to record your thoughts, declutter your mind and keep the things you want to remember. We all only have so much room in our brains for things. Let Linked be the place to put something you want to remember and let you free up space for deeper thinking."
```
Comment (Henryws) [inline packages/linked-deb/linked-deb.pacscript:6]: Can you shorten this any more?

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/linked-deb/linked-deb.pacscript (hunk 1 above)
Comment (soumyatheman) [reply packages/linked-deb/linked-deb.pacscript:6]: I copied it from the official website. If I take the short one, that's too short. 
**"linked - daily journaling without distraction"**
Is this okay?

**Review by Henryws — APPROVED** [review]

**Comment by soumyatheman** [comment]
Merge this please, if everything is okay

**Pull request merged**